runtime/binding-http/src/test/java/io/aklivity/zilla/runtime/binding/http/internal/streams/rfc7540/server/ConnectionManagementIT.java

```diff
@@ -131,12 +131,13 @@ public void httpPostExchangeBeforeSettingsExchange() throws Exception
         k3po.finish();
     }
 
+
     @Test
     @Configuration("server.yaml")
     @Specification({
-        "${net}/http.post.exchange.streaming/client",
-        "${app}/http.post.exchange.streaming/server" })
-    public void httpPostExchangeWhenStreaming() throws Exception
+        "${net}/http.patch.exchange/client",
+        "${app}/http.patch.exchange/server" })
+    public void httpPatchExchange() throws Exception
```

Comment:
I think you accidentally removed `httpPostExchangeWhenStreaming()` method when adding `httpPatchExchange()` method.

Please restore `httpPostExchangeWhenStreaming()` method, so the diff is just an add of `httpPatchExchange()` method, similar to other `ConnectionManagementIT` changes.